spec/resources/schema_dot_org/test.html

```diff
@@ -1,4 +1,4 @@
-<html>    
+<!--<html>    
```

Comment:
you know what, the way i did this the same file is used both when required data is present and missing.  (in the latter case i remove required data AFTER reading in the file.)  now that you pointed this out, mine is a stupid way to do it. i'll create separate files for the various cases.